Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/creating_yml_temp.json

```diff
@@ -0,0 +1,32 @@
+{
+    "resources": [
+        {
```

Comment:
This is for the test? 
If so, put in under the test_data folder. 
Shouldn't be here

Reply:
for the comments above- the new release note in base it's 1.32.0
in crowd_strike_falcon it's 1.10.0.
I run it with demisto release-note

I'll content with you via slack for mini demo

I'll remove the temp file :)